README.md

```diff
@@ -300,7 +300,16 @@ end
 * `cpu_fallback` - Whether to allow libvirt to fall back to a CPU model close
   to the specified model if features in the guest CPU are not supported on the
   host. Defaults to 'allow' if not set. Allowed values: `allow`, `forbid`.
-* `numa_nodes` - Number of NUMA nodes on guest. Must be a factor of `cpu`.
+* `numa_nodes` - Specify an array of NUMA nodes for the guest. The syntax is similar to what would be set in the domain XML. `memory` must be in MB. Symmetrical and asymmetrical topologies are supported but make sure your total count of defined CPUs adds up to `v.cpus`. 
+
+  The sum of all the memory defined here will act as your total memory for your guest VM. **This sum will override what is set in `v.memory`**
+  ```
+  v.cpus = 4
+  v.numa_nodes = [
+    {:id => 0, :cpus => "0-1", :memory => "1024"},
```

Comment:
Looks like id needs to be contiguous range. Tried leaving a hole and libvirt responded with:

Error while creating domain: Error saving the server: Call to virDomainDefineXML failed: XML error: Exactly one 'cell' element per guest NUMA cell allowed, non-contiguous ranges or ranges not starting from 0 are not allowed